bin/update_tarball-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+DEST=/usr/local/repo/tarball-install
```

Comment:
I know nothing about the tarballs! :)

Where does this script come from anyway?  Is it new?

Reply:
Yes, this is a new script. The goal was to automatically retrieve the tarballs from upstream.

I think they're being pushed to repo.osg at present, which doesn't work nicely when there's multiple k8s instances.